upgrade/1.2/README.md

```diff
@@ -9,6 +9,10 @@ for additional reference material in support of the processes and scripts mentio
 
 ## Notes
 
+- **`Important:`** The SMA Grafana service is temporarily inaccessible during the upgrade.
+During stage 3 of the CSM 1.2 upgrade, SMA's Grafana service will become inaccessible at the previous DNS location until the upgrade to SMA 1.6.x is applied. This is due to a change in DNS names for the service.
```

Comment:
```suggestion
During stage 3 of the CSM 1.2 upgrade, the SMA Grafana service will become inaccessible at its previous DNS location. It will
remain inaccessible until the upgrade to SMA 1.6.x is applied. This is because of a change in DNS names for the service.
```